deployment/templates/rabbitmq/rabbitmq-deployment.yaml

```diff
@@ -17,11 +17,21 @@ spec:
       containers:
       - name: rabbitmq
         image: {{ .Values.rabbitmq.image }}
+        command:
+          - /bin/sh
+          - -c
+          - |
+            chmod g-rw /var/lib/rabbitmq/.erlang.cookie;
```

Comment:
What is this for? why is it necessary?

Reply:
I already explained in issue #233 . If rabbitMQ deployment is restarted (e.g. due to cluster failure) rabbitMQ cookie has incorrect permissions after restart ( `rw-rw----` instead of `rw-------`). This is solved by chmod in main container before calling rabbitmq.

Reply:
The idea is to put the information about this workaround in the code itself (deployment/templates/rabbitmq/rabbitmq-deployment.yaml). Otherwise no one will ever find this information.

Reply:
comment added, is it ok now?